### ISSUE TITLE
add some frequently asked questions into F&Q

### ISSUE THREAD

**Issue by lcy-seso** [PaddlePaddle/Paddle#4024]
We are going to select some frequently asked questions into F&Q. I will keep on updating this issue to keep trace of issues we selected.

- [x]  get outputs of multiple layers in inferring: https://github.com/PaddlePaddle/Paddle/issues/4153
- [x] Notes of PaddlePaddle: https://github.com/PaddlePaddle/Paddle/issues/3070
- [x] load pre-trained parameters, formats of the saved parameters in PaddlePaddle. https://github.com/PaddlePaddle/Paddle/issues/2778 https://github.com/PaddlePaddle/Paddle/issues/2812
- [x] how to slove the gradient explosion https://github.com/PaddlePaddle/Paddle/issues/2262
- [x] duplicate layer error https://github.com/PaddlePaddle/Paddle/issues/3615
- [x] learning rate annealing: https://github.com/PaddlePaddle/Paddle/issues/1167#issuecomment-273373637
